transfer.md

```diff
@@ -0,0 +1,49 @@
+# cri-o Usage Transfer
+
+This document outlines useful information for ops and dev transfer as it relates to infrastructure that utilizes cri-o.
+
+## Operational Transfer
+
+## Abstract
+
+The `crio` daemon is intended to provide the [CRI](https://github.com/kubernetes/community/blob/master/contributors/devel/container-runtime-interface.md) socket needed for Kubernetes to use (See the document for [configuring kubernetes to use cri-o](./kubernetes.md) for more information on that).
+Therefore the `crioctl` command line is a client that interfaces to the same grpc socket as the kubernetes daemon would, for talking to the `crio` daemon.
+In many ways `crioctl` is only as feature rich as the Kubernetes CRI requires.
+For this there are additional tools e.g. `kpod` particularly for human interaction.
```

Comment:
suggest dropping "For this" and maybe going with something like:  "There are additional tools e.g. `kpod` and `buildah` that provide a feature rich set of commands for all operational needs in a Kubernetes environment."  Or some such.  Given buildah is mentioned below, I think we should work that in here too in what ever you decide to go with.